### PR TITLE
add import eks using direct method

### PR DESCRIPTION
**Pull request by zryfish**
Add importing EKS using the direct method.

### CODE REVIEW

File: content/en/docs/multicluster-management/import-cloud-hosted-k8s/import-aws-eks.md [hunk 1]
```diff
@@ -1,10 +1,127 @@
...
-keywords: 'kubernetes, kubesphere, multicluster, aws-eks'
+keywords: 'Kubernetes, KubeSphere, multicluster, Amazon eks'
 description: 'Import AWS EKS Cluster'
 
 
 weight: 2340
 ---
 
-TBD
+In this section, we are going to show you how to import EKS to KubeSphere using [`direct connection`](../../enable-multicluster/direct-connection.md) method.
```
Comment (FeynmanZhou) [inline content/en/docs/multicluster-management/import-cloud-hosted-k8s/import-aws-eks.md:10]: All internal links are 404
https://deploy-preview-513--kubesphere-v3.netlify.app/docs/multicluster-management/import-cloud-hosted-k8s/import-aws-eks/

File: content/en/docs/multicluster-management/import-cloud-hosted-k8s/import-aws-eks.md [hunk 2]
```diff
@@ -1,10 +1,127 @@
...
 
 
 weight: 2340
 ---
 
-TBD
+In this section, we are going to show you how to import EKS to KubeSphere using [`direct connection`](../../enable-multicluster/direct-connection.md) method. 
+
+{{< notice note >}}
+If you are planning to import EKS using [`agent connection`](../../enable-multicluster/agent-connection.md), then you can skip this section and follow the [doc](../../enable-multicluster/agent-connection.md) step by step.
```
Comment (FeynmanZhou) [inline content/en/docs/multicluster-management/import-cloud-hosted-k8s/import-aws-eks.md:13]: Same here. Please remove `.md`

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
Please update the internal links

**Comment by Sherlock113** [comment]
The process looks good to me. I will update wording after the pr is merged.

**Pull request merged**